test/MusicStore.E2ETests/DotnetRunTests.cs

```diff
@@ -17,7 +17,7 @@ public class DotnetRunTests : LoggedTest
     {
         public static TestMatrix TestVariants
             => TestMatrix.ForServers(ServerType.Kestrel)
-                .WithTfms(Tfm.NetCoreApp22, Tfm.NetCoreApp21, Tfm.NetCoreApp20, Tfm.Net461);
+                .WithTfms("netcoreapp3.0");
```

Comment:
Should update the TFM list in Hosting.

Reply:
I did. Was feeling impatient and didn't want to wait for 6 hours to get this PR going.

Reply:
Are feature branches still a thing?